cardano-constitution/data/defaultConstitution.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "type": "object",
+    "propertyNames": {
+        "pattern": "^[0-9]+$"
```

Comment:
Nice find. Yes, let's not allow zeroes there, I don't see much of a point. But note that the schema is not enforced in anyway currently, so any JSON integer will be parsed there (because I think json allows leading zeroes)

Reply:
Btw, I didn't want to fix ParamId to be Word, bcs we don't have a word type on Tx and I'd rather we work directly on Integers for easier comparison

Reply:
Oh wait, with your regex we cannot use the number 0 which is an actual protocol parameter

Reply:
I went with this for now: "^(0|[1-9][0-9]*)$"